src/components/memorygame/changeCardStatePropertyToOpposite.jsx

```diff
@@ -0,0 +1,48 @@
+// manipulates isFlipped and isMatched property of card a state
+// property example : "isMatched"
+// cardStates is an array of objects(states)
+// cardStates example :
+// [
+//   {
+//      cardKey:"",
+//      img:{src:"",imgKey:""},
+//      isFlipped:false,
+//      isMatched:false,
+//      description:"",
+//      link:"",
+//      phrase:""
+//   },
+// ...]
+// targetCard is a state obj.
+// Object in line 6-14 is an example for targetCard
```

Comment:
using line numbers in comments guarantees that your comment will be useless when someone else that's not you edits this file, unless you are expecting every teammate to read every comment and look for things like this

Reply:
PTAL